modules.json

```diff
@@ -380,7 +379,7 @@
                     "snpeff/snpeff": {
                         "branch": "master",
                         "git_sha": "911696ea0b62df80e900ef244d7867d177971f73",
-                        "installed_by": ["vcf_annotate_snpeff", "modules"]
+                        "installed_by": ["modules", "vcf_annotate_snpeff"]
```

Comment:
I'm highly disturb by the fact that nf-core tools is changing the order there...

Reply:
still